includes/wikia/services/UserService.class.php

```diff
@@ -3,9 +3,6 @@ class UserService extends Service {
 
 	const CACHE_EXPIRATION = 86400;//1 day
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)